easy-ui-react/src/SearchNav/Title.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from "react";
+import { Text } from "../Text";
+import { classNames } from "../utilities/css";
+
+import styles from "./Title.module.scss";
+
+export type TitleProps = {
+  /**
+   * Emphasized text content to display.
+   */
+  children: ReactNode;
+};
+
+export function Title(props: TitleProps) {
+  const { children } = props;
+
+  return (
+    <span className={classNames(styles.title)}>
+      <Text variant="subtitle1">{children}</Text>
+    </span>
```

Comment:
**question**: how would you feel about adding `transform` support to `Text` so that the intermediary component isn't needed?

something like:

<img width="587" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/d0ca2dca-1bf6-47fd-b8a5-c48ee348dfe5">

<img width="507" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/9f7be0ad-48bd-44a0-a700-4061941ad1c0">

<img width="333" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/ae5475be-066f-4a5c-853f-11efc1cfb9a7">